testing/npd6/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Kevin Daudt <kdaudt@alpinelinux.org>
+# Maintainer: Kevin Daudt <kdaudt@alpinelinux.org>
+pkgname=npd6
+pkgver=1.1.0
+pkgrel=0
+pkgdesc="Neighbor proxy daemon for IPv6"
+url="https://www.ipsidixit.net/"
+arch="all"
+license="GPL-3.0+"
```

Comment:
Should be "GPL-3.0-or-later" according to https://spdx.org/licenses/

Reply:
I accidentally copied it from the deprecated section. Will fix it.